<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1132</width>
    <height>668</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>941</width>
     <height>521</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiCondensed</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/image/baladitech.PNG);
background-image: url(:/image/Design baladitech CRUD.jpg);</string>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Triangular</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideNone</enum>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Gérer les sociétés</string>
    </attribute>
    <widget class="QTabWidget" name="tabWidget_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>851</width>
       <height>441</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Tw Cen MT</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/image/Design baladitech inCRUD.png);</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Ajouter</string>
      </attribute>
      <widget class="QLineEdit" name="lineEdit_13">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>210</y>
         <width>251</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_32">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>120</y>
         <width>171</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>ID_société:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_12">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>120</y>
         <width>251</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_31">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>210</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Nom société:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>550</x>
         <y>160</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Confirmer</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Afficher</string>
      </attribute>
      <widget class="QLabel" name="label_35">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>300</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Trier par:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_37">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>320</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Rechercher:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_17">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>310</y>
         <width>231</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_7">
       <property name="geometry">
        <rect>
         <x>720</x>
         <y>310</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Rechercher</string>
       </property>
      </widget>
      <widget class="QTableView" name="Affichage_2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>10</y>
         <width>521</width>
         <height>281</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_5">
       <property name="geometry">
        <rect>
         <x>590</x>
         <y>90</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Modifier</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>590</x>
         <y>160</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>310</y>
         <width>97</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Nom</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_2">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>350</y>
         <width>97</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="Ajouter">
    <attribute name="title">
     <string>Gérer les chantiers</string>
    </attribute>
    <widget class="QTabWidget" name="tabWidget_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>851</width>
       <height>441</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Tw Cen MT</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/image/Design baladitech inCRUD.png);</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_8">
      <attribute name="title">
       <string>Ajouter</string>
      </attribute>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>35</x>
         <y>70</y>
         <width>171</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Nom chantier:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>130</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Localisation:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>70</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>110</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Régions</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>110</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Rue/avenue</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_2">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>140</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Region nord</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Region sud</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Region est</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Region ouest</string>
        </property>
       </item>
      </widget>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>140</y>
         <width>113</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>210</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Affecter à société:</string>
       </property>
      </widget>
      <widget class="QCommandLinkButton" name="commandLinkButton">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>330</y>
         <width>187</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Nouvelle société</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_10">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>20</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
      <widget class="QLabel" name="label_27">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>20</y>
         <width>161</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>ID_chantier:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>340</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Confirmer</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_29">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>280</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>état:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>281</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>En attente</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>En cours</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Terminé</string>
        </property>
       </item>
      </widget>
      <widget class="QTableView" name="Affichage_4">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>90</y>
         <width>281</width>
         <height>201</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>210</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_10">
      <attribute name="title">
       <string>Afficher</string>
      </attribute>
      <widget class="QLabel" name="label_26">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>320</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Trier par:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_38">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>330</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Rechercher:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_8">
       <property name="geometry">
        <rect>
         <x>700</x>
         <y>330</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Rechercher</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_18">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>330</y>
         <width>221</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_6">
       <property name="geometry">
        <rect>
         <x>730</x>
         <y>190</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Modifier</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_11">
       <property name="geometry">
        <rect>
         <x>730</x>
         <y>240</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
      <widget class="QTableView" name="Affichage_3">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>691</width>
         <height>281</height>
        </rect>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_3">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>320</y>
         <width>97</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Nom</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_4">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>349</y>
         <width>97</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Region</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_5">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>380</y>
         <width>97</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Etat</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
